### ISSUE TITLE
traefik v2.x prometheus metrics

### ISSUE THREAD

**Issue by DeoLeung** [containous/traefik#6560]
### Do you want to request a *feature* or report a *bug*?

Feature

### What did you expect to see?

at the moment I can see `v2.x` expose following metrics to prometheus

* traefik_service_requests_total
* traefik_config_last_reload_failure
* traefik_entrypoint_request_duration_seconds_count
* traefik_service_request_duration_seconds_sum
* traefik_config_reloads_total
* traefik_entrypoint_request_duration_seconds_sum
* traefik_service_open_connections
* traefik_entrypoint_requests_total
* traefik_service_request_duration_seconds_count
* traefik_config_last_reload_success
* traefik_config_reloads_failure_total
* traefik_entrypoint_request_duration_seconds_bucket
* traefik_entrypoint_open_connections
* traefik_service_request_duration_seconds_bucket

I would like to request some features
 *  metrics like `server_up` in `v1.x` so I can count the active servers
 * flag to enable adding `url` label to `traefik_service_request_duration_seconds` so we can do a bit more detail observation on that
 * with `docker` provider, since traefik knows the address of the service servers, would be great if traefik support exposing it to `prometheus`

**Comment by zolech** [comment]
+1 for `server_up` metric

I would like to see that as well. Right now I had to remove this from my dashboards after migrating from Marathon to k8s provider. I see this information in traefik dashboard but there is no such thing in metrics.

**Comment by thewilli** [comment]
having the amount of days until a used tls certificate is about to expire would be useful as well

**Comment by Kiina** [comment]
Adding to the tls certificate, how about seeing handshake informaton like cipherSuites and tls protocol versions for connections? It would be kinda nice to know how many connections aren't made due to the client only supporting low tls versions or weak ciphers.

**Comment by rcomanne** [comment]
Yes, +1 for the `server_up` metrics, we use this for monitoring our services, losing this metric means waiting with upgrading.
Any idea if this will be added anytime soon?

**Comment by mgerasimchuk** [comment]
Recently I moved from ingress-nginx controller to traefik, and only now I realised, that traefik unfortunately doesn't have any way to collect metrics by URL path. For example in ingress-nginx controller this behavior works by default: https://github.com/kubernetes/ingress-nginx/blob/96b6228a6b65a85e421b8a348a149e99181664d1/deploy/grafana/dashboards/request-handling-performance.json#L314

Traefik has a lot of pros like SSL certificates management and very comfortable routing, but observability of the system also is one of the most important thing, and now I can't understand which route of my API works slow....
Hope that the path label will be added in nearest future..
